enterprise/server/configure.md

```diff
@@ -315,6 +315,41 @@ ephox {
 }
 ```
 
+### `spelling` (optional)
+
+The Spelling service has two configurable settings:
+
+- `custom-dictionaries-path`
+- `dynamic-custom-dictionaries`
+
+#### `custom-dictionaries-path` (optional)
+
+Used to define the location of the custom dictionaries. When the setting is not provided, no custom dictionaries are loaded.
+
+For example:
+
+```
+ephox {
+  spelling {
+    custom-dictionaries-path = "/opt/ephox/dictionaries"
+  }
+}
+```
+
+#### `dynamic-custom-dictionaries` (optional)
```

Comment:
Should this be linked to the other page, or vice versa?

Reply:
Yeah, I'm not sure. I think the information has a different focus, but I would just be guessing.

Reply:
yeah nah. Looking at these two pages, I _think_ the expectation was that users would look at the custom spelling page rather than this one for this information, which I do not agree with.

I've de-duped the information by using includes (commit WIP).

Reply:
de-duped here: [`314b1ce` (#2011)](https://github.com/tinymce/tinymce-docs/pull/2011/commits/314b1ceadb2bec63e8102a9e59c11a120ef64875)